test/JDBC/input/BABEL-3117-vu-prepare.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE BABEL_3117_prepare_db1
+CREATE DATABASE BABEL_3117_prepare_db2
+GO
+
+USE BABEL_3117_prepare_db1
+GO
+CREATE TABLE BABEL_3117_prepare_db1_employeeData(ID INT IDENTITY (1,1) PRIMARY KEY,Emp_First_name VARCHAR (50))
```

Comment:
Code has lot of schema handling (explicit and implicit), can we have test cases to cover all use cases

Reply:
+1 . Can you please review your test cases with Xiaohui to confirm that we have not missed any cases?